src/models/todo.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseModel } from "./base-model";
+import { LooseObject } from "@typings";
+
+export class Todo extends BaseModel {
```

Comment:
Should we use `TodoItem` instead of `Todo` to make it a noun instead of a verb?

Reply:
ya sure